### PR TITLE
Test/161 unit tests team list components

### PR DESCRIPTION
**Pull request by danielevalverde**

### CODE REVIEW

File: src/app/components/team/TeamLists/TeamListsColumn.test.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  const team = {
+    slug: 'new-team',
+    id: '1',
+    team_bot_installations: {
+      edges: [],
+    },
+  };
+
+  it('should render collum', () => {
```
Comment (caiosba) [inline src/app/components/team/TeamLists/TeamListsColumn.test.js:26]: Typo on `collum`

File: src/app/components/team/TeamLists/TeamListsColumn.test.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    const wrapper = mountWithIntl(<TeamListsColumn
+      title="Colum title"
```
Comment (caiosba) [inline src/app/components/team/TeamLists/TeamListsColumn.test.js:28]: Typo on `Colum`

File: src/app/components/team/TeamLists/TeamListsColumn.test.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      team={team}
+      onToggle={() => {}}
+    />);
+    expect(wrapper.find('.MuiTypography-subtitle2').hostNodes()).toHaveLength(1);
+    expect(wrapper.html()).toMatch('Colum title');
+    expect(wrapper.html()).toMatch('label-content-1');
+    expect(wrapper.html()).toMatch('label-content-2');
+    expect(wrapper.html()).toMatch('label-content-3');
+  });
+});
```
Comment (caiosba) [inline src/app/components/team/TeamLists/TeamListsColumn.test.js:39]: Would be great to test this condition: https://github.com/meedan/check-web/blob/develop/src/app/components/team/TeamLists/TeamListsColumn.js#L56-L59

This means:

- In the test above, make sure that the "placeholder" doesn't appear
- Add another test where `columns` is empty and make sure that the placeholder appears

File: src/app/components/team/TeamLists/TeamListsItem.test.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    show: false,
+  };
+
+  it('render item and hide button', () => {
+    const wrapper = mountWithIntl(<TeamListsItem
+      team={team}
+      column={column}
+      onToggle={() => {}}
+    />);
+    expect(wrapper.html()).toMatch('Hide');
```
Comment (caiosba) [inline src/app/components/team/TeamLists/TeamListsItem.test.js:31]: I would add this here too: `expect(wrapper.html()).not.toMatch('Show');`

File: src/app/components/team/TeamLists/TeamListsItem.test.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    expect(wrapper.find('#team-lists__item-1-key-content').hostNodes()).toHaveLength(1);
+  });
+
+  it('render item and show button', () => {
+    const wrapper = mountWithIntl(<TeamListsItem
+      team={team}
+      column={column2}
+      onToggle={() => {}}
+    />);
+    expect(wrapper.html()).toMatch('Show');
```
Comment (caiosba) [inline src/app/components/team/TeamLists/TeamListsItem.test.js:43]: I would add this here too: `expect(wrapper.html()).not.toMatch('Hide');`

File: src/app/components/team/TeamLists/TeamListsItem.test.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    expect(wrapper.html()).toMatch('label-content');
+    expect(wrapper.html()).toMatch('General');
+    expect(wrapper.find('#team-lists__item-2-key-content').hostNodes()).toHaveLength(1);
+  });
+});
```
Comment (caiosba) [inline src/app/components/team/TeamLists/TeamListsItem.test.js:48]: Other cases that would be good to test:

1) Arrows are displayed only if both `onMoveUp` and `onMoveDown` are provided (https://github.com/meedan/check-web/blob/develop/src/app/components/team/TeamLists/TeamListsItem.js#L68)

2) Metadata label is displayed for metadata columns (https://github.com/meedan/check-web/blob/develop/src/app/components/team/TeamLists/TeamListsItem.js#L90)

**Review by caiosba — CHANGES_REQUESTED** [review]
Please take a look at my inline comments - overall, great work!

**Review by amoedoamorim — APPROVED** [review]
